ogr/services/gitlab/issue.py

```diff
@@ -47,7 +47,7 @@ def url(self) -> str:
         return self._raw_issue.web_url
 
     @property
-    def assignees(self) -> list:
+    def assignees(self) -> Optional[list]:
```

Comment:
```suggestion
    def assignees(self) -> Optional[List[???]]:
```

I'm surprised mypy isn't yelling :D

I hope for `s/???/str`, but it's GitLab, so I'm not willing to bet on it :upside_down_face: